local環境でテストを実行すると失敗する

npm testを実行するとテストが全て失敗した。

 Error: Cannot find module 'typescript'

おそらく以前はlocal環境にtypescriptが入っていたが、今ははいっていないためと思われる。
しかし、それならば、なぜgithub actionsのテストは成功するのだろうか

その後、npm install -D typescriptしたら、テストが成功した。
その後、変更をもとに戻してもテストが成功した。何かしらのパッケージがglobalなtypescriptを見ているのだろうか？

テスト実行時にエラーが発生したことは真実。
どこかのタイミングでtypescriptがinstallされればエラーは発生しないようだ。
ということで、typescriptをdev-dependenciesに追加する。

<details>
<summary>error log</summary>
❯ npm test    

> eslint-plugin-mui-path-imports@0.0.3 test
> mocha tests --recursive



  rule
    valid
      1) import Box from "@mui/material/Box";
      2) import Add from "@mui/icons-material/Add";
      3) import { Theme } from "@mui/material";
      4) import { SvgIconTypeMap } from "@mui/material";
      5) import { SvgIconTypeMap, Theme } from "@mui/material";
      6) import { Theme, SvgIconTypeMap } from "@mui/material";
      7) import { Other } from "other/package";
    invalid
      8) import { Add } from "@mui/icons-material";
      9) import { Box } from "@mui/material";
      10) import { Add, Link } from "@mui/icons-material";
      11) import { Box, Card } from "@mui/material";
      12) import { Box, Theme } from "@mui/material";
      13) import { Box, Theme, SvgIconTypeMap } from "@mui/material";
      14) import { Theme, Box, SvgIconTypeMap } from "@mui/material";
      15) import { Theme, SvgIconTypeMap, Box } from "@mui/material";


  0 passing (97ms)
  15 failing

  1) rule
       valid
         import Box from "@mui/material/Box";:
     Error: Cannot find module 'typescript'
Require stack:
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/typescript-estree/dist/parser.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/typescript-estree/dist/index.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/parser/dist/parser.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/parser/dist/index.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/eslint/lib/rule-tester/rule-tester.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/eslint/lib/rule-tester/index.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/eslint/lib/api.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/tests/lib/rules/mui-path-imports.js
      at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
      at Function.Module._load (node:internal/modules/cjs/loader:778:27)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/typescript-estree/dist/parser.js:35:25)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/typescript-estree/dist/index.js:18:16)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/parser/dist/parser.js:7:29)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/parser/dist/index.js:4:16)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at runRuleForItem (node_modules/eslint/lib/rule-tester/rule-tester.js:608:59)
      at testValidTemplate (node_modules/eslint/lib/rule-tester/rule-tester.js:704:28)
      at Context.<anonymous> (node_modules/eslint/lib/rule-tester/rule-tester.js:986:29)
      at processImmediate (node:internal/timers:466:21)

  2) rule
       valid
         import Add from "@mui/icons-material/Add";:
     Error: Cannot find module 'typescript'
Require stack:
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/typescript-estree/dist/parser.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/typescript-estree/dist/index.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/parser/dist/parser.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/parser/dist/index.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/eslint/lib/rule-tester/rule-tester.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/eslint/lib/rule-tester/index.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/eslint/lib/api.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/tests/lib/rules/mui-path-imports.js
      at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
      at Function.Module._load (node:internal/modules/cjs/loader:778:27)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/typescript-estree/dist/parser.js:35:25)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/typescript-estree/dist/index.js:18:16)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/parser/dist/parser.js:7:29)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/parser/dist/index.js:4:16)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at runRuleForItem (node_modules/eslint/lib/rule-tester/rule-tester.js:608:59)
      at testValidTemplate (node_modules/eslint/lib/rule-tester/rule-tester.js:704:28)
      at Context.<anonymous> (node_modules/eslint/lib/rule-tester/rule-tester.js:986:29)
      at processImmediate (node:internal/timers:466:21)

  3) rule
       valid
         import { Theme } from "@mui/material";:
     Error: Cannot find module 'typescript'
Require stack:
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/typescript-estree/dist/parser.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/typescript-estree/dist/index.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/parser/dist/parser.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/parser/dist/index.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/eslint/lib/rule-tester/rule-tester.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/eslint/lib/rule-tester/index.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/eslint/lib/api.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/tests/lib/rules/mui-path-imports.js
      at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
      at Function.Module._load (node:internal/modules/cjs/loader:778:27)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/typescript-estree/dist/parser.js:35:25)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/typescript-estree/dist/index.js:18:16)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/parser/dist/parser.js:7:29)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/parser/dist/index.js:4:16)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at runRuleForItem (node_modules/eslint/lib/rule-tester/rule-tester.js:608:59)
      at testValidTemplate (node_modules/eslint/lib/rule-tester/rule-tester.js:704:28)
      at Context.<anonymous> (node_modules/eslint/lib/rule-tester/rule-tester.js:986:29)
      at processImmediate (node:internal/timers:466:21)

  4) rule
       valid
         import { SvgIconTypeMap } from "@mui/material";:
     Error: Cannot find module 'typescript'
Require stack:
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/typescript-estree/dist/parser.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/typescript-estree/dist/index.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/parser/dist/parser.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/parser/dist/index.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/eslint/lib/rule-tester/rule-tester.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/eslint/lib/rule-tester/index.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/eslint/lib/api.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/tests/lib/rules/mui-path-imports.js
      at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
      at Function.Module._load (node:internal/modules/cjs/loader:778:27)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/typescript-estree/dist/parser.js:35:25)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/typescript-estree/dist/index.js:18:16)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/parser/dist/parser.js:7:29)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/parser/dist/index.js:4:16)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at runRuleForItem (node_modules/eslint/lib/rule-tester/rule-tester.js:608:59)
      at testValidTemplate (node_modules/eslint/lib/rule-tester/rule-tester.js:704:28)
      at Context.<anonymous> (node_modules/eslint/lib/rule-tester/rule-tester.js:986:29)
      at processImmediate (node:internal/timers:466:21)

  5) rule
       valid
         import { SvgIconTypeMap, Theme } from "@mui/material";:
     Error: Cannot find module 'typescript'
Require stack:
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/typescript-estree/dist/parser.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/typescript-estree/dist/index.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/parser/dist/parser.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/parser/dist/index.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/eslint/lib/rule-tester/rule-tester.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/eslint/lib/rule-tester/index.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/eslint/lib/api.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/tests/lib/rules/mui-path-imports.js
      at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
      at Function.Module._load (node:internal/modules/cjs/loader:778:27)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/typescript-estree/dist/parser.js:35:25)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/typescript-estree/dist/index.js:18:16)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/parser/dist/parser.js:7:29)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/parser/dist/index.js:4:16)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at runRuleForItem (node_modules/eslint/lib/rule-tester/rule-tester.js:608:59)
      at testValidTemplate (node_modules/eslint/lib/rule-tester/rule-tester.js:704:28)
      at Context.<anonymous> (node_modules/eslint/lib/rule-tester/rule-tester.js:986:29)
      at processImmediate (node:internal/timers:466:21)

  6) rule
       valid
         import { Theme, SvgIconTypeMap } from "@mui/material";:
     Error: Cannot find module 'typescript'
Require stack:
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/typescript-estree/dist/parser.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/typescript-estree/dist/index.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/parser/dist/parser.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/parser/dist/index.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/eslint/lib/rule-tester/rule-tester.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/eslint/lib/rule-tester/index.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/eslint/lib/api.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/tests/lib/rules/mui-path-imports.js
      at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
      at Function.Module._load (node:internal/modules/cjs/loader:778:27)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/typescript-estree/dist/parser.js:35:25)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/typescript-estree/dist/index.js:18:16)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/parser/dist/parser.js:7:29)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/parser/dist/index.js:4:16)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at runRuleForItem (node_modules/eslint/lib/rule-tester/rule-tester.js:608:59)
      at testValidTemplate (node_modules/eslint/lib/rule-tester/rule-tester.js:704:28)
      at Context.<anonymous> (node_modules/eslint/lib/rule-tester/rule-tester.js:986:29)
      at processImmediate (node:internal/timers:466:21)

  7) rule
       valid
         import { Other } from "other/package";:
     Error: Cannot find module 'typescript'
Require stack:
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/typescript-estree/dist/parser.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/typescript-estree/dist/index.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/parser/dist/parser.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/parser/dist/index.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/eslint/lib/rule-tester/rule-tester.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/eslint/lib/rule-tester/index.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/eslint/lib/api.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/tests/lib/rules/mui-path-imports.js
      at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
      at Function.Module._load (node:internal/modules/cjs/loader:778:27)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/typescript-estree/dist/parser.js:35:25)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/typescript-estree/dist/index.js:18:16)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/parser/dist/parser.js:7:29)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/parser/dist/index.js:4:16)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at runRuleForItem (node_modules/eslint/lib/rule-tester/rule-tester.js:608:59)
      at testValidTemplate (node_modules/eslint/lib/rule-tester/rule-tester.js:704:28)
      at Context.<anonymous> (node_modules/eslint/lib/rule-tester/rule-tester.js:986:29)
      at processImmediate (node:internal/timers:466:21)

  8) rule
       invalid
         import { Add } from "@mui/icons-material";:
     Error: Cannot find module 'typescript'
Require stack:
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/typescript-estree/dist/parser.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/typescript-estree/dist/index.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/parser/dist/parser.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/parser/dist/index.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/eslint/lib/rule-tester/rule-tester.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/eslint/lib/rule-tester/index.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/eslint/lib/api.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/tests/lib/rules/mui-path-imports.js
      at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
      at Function.Module._load (node:internal/modules/cjs/loader:778:27)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/typescript-estree/dist/parser.js:35:25)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/typescript-estree/dist/index.js:18:16)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/parser/dist/parser.js:7:29)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/parser/dist/index.js:4:16)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at runRuleForItem (node_modules/eslint/lib/rule-tester/rule-tester.js:608:59)
      at testInvalidTemplate (node_modules/eslint/lib/rule-tester/rule-tester.js:758:28)
      at Context.<anonymous> (node_modules/eslint/lib/rule-tester/rule-tester.js:997:29)
      at processImmediate (node:internal/timers:466:21)

  9) rule
       invalid
         import { Box } from "@mui/material";:
     Error: Cannot find module 'typescript'
Require stack:
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/typescript-estree/dist/parser.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/typescript-estree/dist/index.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/parser/dist/parser.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/parser/dist/index.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/eslint/lib/rule-tester/rule-tester.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/eslint/lib/rule-tester/index.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/eslint/lib/api.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/tests/lib/rules/mui-path-imports.js
      at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
      at Function.Module._load (node:internal/modules/cjs/loader:778:27)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/typescript-estree/dist/parser.js:35:25)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/typescript-estree/dist/index.js:18:16)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/parser/dist/parser.js:7:29)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/parser/dist/index.js:4:16)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at runRuleForItem (node_modules/eslint/lib/rule-tester/rule-tester.js:608:59)
      at testInvalidTemplate (node_modules/eslint/lib/rule-tester/rule-tester.js:758:28)
      at Context.<anonymous> (node_modules/eslint/lib/rule-tester/rule-tester.js:997:29)
      at processImmediate (node:internal/timers:466:21)

  10) rule
       invalid
         import { Add, Link } from "@mui/icons-material";:
     Error: Cannot find module 'typescript'
Require stack:
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/typescript-estree/dist/parser.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/typescript-estree/dist/index.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/parser/dist/parser.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/parser/dist/index.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/eslint/lib/rule-tester/rule-tester.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/eslint/lib/rule-tester/index.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/eslint/lib/api.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/tests/lib/rules/mui-path-imports.js
      at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
      at Function.Module._load (node:internal/modules/cjs/loader:778:27)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/typescript-estree/dist/parser.js:35:25)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/typescript-estree/dist/index.js:18:16)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/parser/dist/parser.js:7:29)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/parser/dist/index.js:4:16)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at runRuleForItem (node_modules/eslint/lib/rule-tester/rule-tester.js:608:59)
      at testInvalidTemplate (node_modules/eslint/lib/rule-tester/rule-tester.js:758:28)
      at Context.<anonymous> (node_modules/eslint/lib/rule-tester/rule-tester.js:997:29)
      at processImmediate (node:internal/timers:466:21)

  11) rule
       invalid
         import { Box, Card } from "@mui/material";:
     Error: Cannot find module 'typescript'
Require stack:
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/typescript-estree/dist/parser.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/typescript-estree/dist/index.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/parser/dist/parser.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/parser/dist/index.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/eslint/lib/rule-tester/rule-tester.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/eslint/lib/rule-tester/index.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/eslint/lib/api.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/tests/lib/rules/mui-path-imports.js
      at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
      at Function.Module._load (node:internal/modules/cjs/loader:778:27)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/typescript-estree/dist/parser.js:35:25)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/typescript-estree/dist/index.js:18:16)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/parser/dist/parser.js:7:29)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/parser/dist/index.js:4:16)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at runRuleForItem (node_modules/eslint/lib/rule-tester/rule-tester.js:608:59)
      at testInvalidTemplate (node_modules/eslint/lib/rule-tester/rule-tester.js:758:28)
      at Context.<anonymous> (node_modules/eslint/lib/rule-tester/rule-tester.js:997:29)
      at processImmediate (node:internal/timers:466:21)

  12) rule
       invalid
         import { Box, Theme } from "@mui/material";:
     Error: Cannot find module 'typescript'
Require stack:
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/typescript-estree/dist/parser.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/typescript-estree/dist/index.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/parser/dist/parser.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/parser/dist/index.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/eslint/lib/rule-tester/rule-tester.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/eslint/lib/rule-tester/index.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/eslint/lib/api.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/tests/lib/rules/mui-path-imports.js
      at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
      at Function.Module._load (node:internal/modules/cjs/loader:778:27)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/typescript-estree/dist/parser.js:35:25)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/typescript-estree/dist/index.js:18:16)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/parser/dist/parser.js:7:29)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/parser/dist/index.js:4:16)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at runRuleForItem (node_modules/eslint/lib/rule-tester/rule-tester.js:608:59)
      at testInvalidTemplate (node_modules/eslint/lib/rule-tester/rule-tester.js:758:28)
      at Context.<anonymous> (node_modules/eslint/lib/rule-tester/rule-tester.js:997:29)
      at processImmediate (node:internal/timers:466:21)

  13) rule
       invalid
         import { Box, Theme, SvgIconTypeMap } from "@mui/material";:
     Error: Cannot find module 'typescript'
Require stack:
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/typescript-estree/dist/parser.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/typescript-estree/dist/index.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/parser/dist/parser.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/parser/dist/index.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/eslint/lib/rule-tester/rule-tester.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/eslint/lib/rule-tester/index.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/eslint/lib/api.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/tests/lib/rules/mui-path-imports.js
      at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
      at Function.Module._load (node:internal/modules/cjs/loader:778:27)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/typescript-estree/dist/parser.js:35:25)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/typescript-estree/dist/index.js:18:16)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/parser/dist/parser.js:7:29)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/parser/dist/index.js:4:16)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at runRuleForItem (node_modules/eslint/lib/rule-tester/rule-tester.js:608:59)
      at testInvalidTemplate (node_modules/eslint/lib/rule-tester/rule-tester.js:758:28)
      at Context.<anonymous> (node_modules/eslint/lib/rule-tester/rule-tester.js:997:29)
      at processImmediate (node:internal/timers:466:21)

  14) rule
       invalid
         import { Theme, Box, SvgIconTypeMap } from "@mui/material";:
     Error: Cannot find module 'typescript'
Require stack:
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/typescript-estree/dist/parser.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/typescript-estree/dist/index.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/parser/dist/parser.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/parser/dist/index.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/eslint/lib/rule-tester/rule-tester.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/eslint/lib/rule-tester/index.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/eslint/lib/api.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/tests/lib/rules/mui-path-imports.js
      at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
      at Function.Module._load (node:internal/modules/cjs/loader:778:27)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/typescript-estree/dist/parser.js:35:25)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/typescript-estree/dist/index.js:18:16)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/parser/dist/parser.js:7:29)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/parser/dist/index.js:4:16)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at runRuleForItem (node_modules/eslint/lib/rule-tester/rule-tester.js:608:59)
      at testInvalidTemplate (node_modules/eslint/lib/rule-tester/rule-tester.js:758:28)
      at Context.<anonymous> (node_modules/eslint/lib/rule-tester/rule-tester.js:997:29)
      at processImmediate (node:internal/timers:466:21)

  15) rule
       invalid
         import { Theme, SvgIconTypeMap, Box } from "@mui/material";:
     Error: Cannot find module 'typescript'
Require stack:
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/typescript-estree/dist/parser.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/typescript-estree/dist/index.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/parser/dist/parser.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/@typescript-eslint/parser/dist/index.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/eslint/lib/rule-tester/rule-tester.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/eslint/lib/rule-tester/index.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/node_modules/eslint/lib/api.js
- /home/kajiri/ghq/github.com/kajirikajiri/eslint-plugin-mui-path-imports/tests/lib/rules/mui-path-imports.js
      at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
      at Function.Module._load (node:internal/modules/cjs/loader:778:27)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/typescript-estree/dist/parser.js:35:25)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/typescript-estree/dist/index.js:18:16)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/parser/dist/parser.js:7:29)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at Object.<anonymous> (node_modules/@typescript-eslint/parser/dist/index.js:4:16)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Module.require (node:internal/modules/cjs/loader:1005:19)
      at require (node:internal/modules/cjs/helpers:102:18)
      at runRuleForItem (node_modules/eslint/lib/rule-tester/rule-tester.js:608:59)
      at testInvalidTemplate (node_modules/eslint/lib/rule-tester/rule-tester.js:758:28)
      at Context.<anonymous> (node_modules/eslint/lib/rule-tester/rule-tester.js:997:29)
      at processImmediate (node:internal/timers:466:21)

</details>

--- a/package.json
+++ b/package.json
@@ -29,7 +29,8 @@
     "eslint": "^8.0.1",
     "eslint-plugin-eslint-plugin": "^4.0.1",
     "eslint-plugin-node": "^11.1.0",
-    "mocha": "^9.1.3"
+    "mocha": "^9.1.3",
+    "typescript": "^4.9.3"
   },
   "engines": {
     "node": "12.x || 14.x || >= 16"

